samples/attested_tls/non_enc_client/cert_verify_config.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef ATTLESTED_TLS_ENCLAVE_CONFIG
+#define ATTLESTED_TLS_ENCLAVE_CONFIG
```

Comment:
Aside: should we consider using `#pragma once` throughout instead of these include guards?
https://en.wikipedia.org/wiki/Pragma_once#Portability shows the pragma is supported by gcc, clang, MSVC, and many others.

Reply:
@CodeMonkeyLeet @anakrish Do you have any comments on this?